### PR TITLE
add eligibility to ccdf_subsidy formula

### LINKED ISSUE #476: Require eligibility in CCDF subsidy
`spm_unit_ccdf_subsidy` needs to be based on child-level `is_ccdf_eligible`.

I'd suggest also removing `spm_unit_total_childcare_market_rate` which can be folded into the `spm_unit_ccdf_subsidy` formula.

### PR DESCRIPTION
**Pull request by tolaouk**
fixes #476

### CODE REVIEW

File: openfisca_us/tests/policy/baseline/hhs/ccdf/spm_unit_ccdf_subsidy.yaml [hunk 1]
```diff
@@ -30,3 +30,18 @@
...
     spm_unit_ccdf_subsidy: 12_650
+
+- name: 1 child in DCC/SACC provider, Queens county, NY state, family income $50,000, family size 3, family asset above $1,000,000
+  period: 2022
+  absolute_error_margin: 0.01
+  input:
+    people:
+      child:
+        ccdf_market_rate: 21_112
+        is_ccdf_eligible: FALSE
```
Comment (MaxGhenis) [inline openfisca_us/tests/policy/baseline/hhs/ccdf/spm_unit_ccdf_subsidy.yaml:41]: and elsewhere - see #447
```suggestion
        is_ccdf_eligible: false
```

File: openfisca_us/tests/policy/baseline/hhs/ccdf/spm_unit_ccdf_subsidy.yaml [hunk 2]
```diff
@@ -1,7 +1,47 @@
...
+    people:
+      child:
+        ccdf_market_rate: 21_112
+        is_ccdf_eligible: false
+    spm_units:
+      spm_unit:
+        members: [child]
+        spm_unit_total_ccdf_copay: 2_804
+  output:
+    spm_unit_ccdf_subsidy: 0
```
Comment (MaxGhenis) [inline openfisca_us/tests/policy/baseline/hhs/ccdf/spm_unit_ccdf_subsidy.yaml:47]: Could just add another test with eligible children whose copay exceeds their market rate to check the subsidy is zero? After that looks good to merge.
